streaming_data_types/array_1d_se00.py

```diff
@@ -0,0 +1,159 @@
+from datetime import datetime, timezone
+from typing import List, NamedTuple, Optional, Union
+
+import flatbuffers
+import numpy as np
+
+from streaming_data_types.fbschemas.array_1d_se00.Location import Location
+from streaming_data_types.fbschemas.array_1d_se00.SampleEnvironmentData import (
+    SampleEnvironmentData,
+    SampleEnvironmentDataAddChannel,
+    SampleEnvironmentDataAddMessageCounter,
+    SampleEnvironmentDataAddName,
+    SampleEnvironmentDataAddPacketTimestamp,
+    SampleEnvironmentDataAddTimeDelta,
+    SampleEnvironmentDataAddTimestampLocation,
+    SampleEnvironmentDataAddTimestamps,
+    SampleEnvironmentDataAddValues,
+    SampleEnvironmentDataAddValuesType,
+    SampleEnvironmentDataEnd,
+    SampleEnvironmentDataStart,
+)
+from streaming_data_types.fbschemas.array_1d_se00.ValueUnion import ValueUnion
+from streaming_data_types.utils import check_schema_identifier
+
+FILE_IDENTIFIER = b"se00"
+
+
+def serialise_se00(
+    name: str,
+    channel: int,
+    timestamp: datetime,
```

Comment:
Sorry, I didn't notice this before but it would be more consistent to pass in the timestamp as ns since the epoch rather than datetime
Then it doesn't require the messy `int(timestamp.timestamp() * 1e9)` below as the caller can just pass in `time.time_ns()`.